File: src/cs/file_read.cs
class Program
{
	public static void Main()
	{
        System.Console.WriteLine(System.IO.File.ReadAllText("../files/content.txt"));

        foreach (var line in System.IO.File.ReadAllLines("../files/multipleLines.txt"))
            System.Console.WriteLine($"- {line}");
	}
}

File: src/vb/file_read.vb
Public Module Program
    Sub Main()
        System.Console.WriteLine(System.IO.File.ReadAllText("../files/content.txt"))

        For Each line As String In System.IO.File.ReadAllLines("../files/multipleLines.txt")
            System.Console.WriteLine("- " & line)
        Next
    End Sub
End Module